operator.yml

```diff
@@ -33,7 +33,7 @@ spec:
     served: true
     storage: true
 ---
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
How does the `apiVersion` change to `beta1` contribute to the solution?  Just curious?

Reply:
RBAC went to v1 in 1.8, so for 3.7 clusters they would only have the 'v1beta1' endpoint.

Assuming that the v1beta1 endpoint is still present on newer versions so by using that endpoint could be way to have same code work with rbac on 3.7+